### PR TITLE
fix(css): FRMW-421-Remove deprecated .title from rxActionMenu

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-421

- [x] Dev LGTM
- [x] Dev LGTM

---
Remove deprecated CSS `.title` class from `rxActionMenu`.

### CODE REVIEW

File: src/rxActionMenu/docs/rxActionMenu.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
     by clicking on the cog itself.
     </p>
 
-    <h3 class="title">Typical Usage</h3>
+    <h3 class="midway-test-title">Typical Usage</h3>
```
Comment (CITguy) [inline src/rxActionMenu/docs/rxActionMenu.html]: I'd go with changing to an ID.

```html
<h3 id="typical-usage">Typical Usage</h3>
```

Suggested update to midway below.

File: src/rxActionMenu/docs/rxActionMenu.midway.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ describe('rxActionMenu', function () {
     };
 
     var clickSomewhereElse = function () {
-        $('.component-demo .title').click();
+        $('.component-demo .midway-test-title').click();
```
Comment (CITguy) [inline src/rxActionMenu/docs/rxActionMenu.midway.js]: (with suggested update above)

```javascript
$('#typical-usage').click();
```

**Comment by 100stacks** [comment]
@CITguy comments incorporated. :+1:

**Comment by bry4n** [comment]
LGTM

**Comment by CITguy** [comment]
LGTM

**Pull request merged**